src/tracks/playabletrack/wavetrack/ui/WaveClipTrimHandle.h

```diff
@@ -29,20 +29,31 @@ class WaveClipTrimHandle : public UIHandle
        virtual ~ClipTrimPolicy();
 
        virtual bool Init(const TrackPanelMouseEvent& event) = 0;
-       virtual void Trim(const TrackPanelMouseEvent& event, AudacityProject& project) = 0;
+       virtual UIHandle::Result Trim(const TrackPanelMouseEvent& event, AudacityProject& project) = 0;
        virtual void Finish(AudacityProject& project) = 0;
        virtual void Cancel() = 0;
+
+       virtual void Draw(
+           TrackPanelDrawingContext &context, 
+           const wxRect &rect, 
+           unsigned iPass);
+
+       virtual wxRect DrawingArea(
+           TrackPanelDrawingContext&, 
+           const wxRect &rect, 
+           const wxRect &panelRect, 
+           unsigned iPass);
```

Comment:
One of the C++ Core Guidelines I like is that each virtual function declaration should be marked with exactly one of `virtual` `override` or `final` as appropriate, and for these two it should be `override`.

Reply:
Oh, I see I misunderstood the context.  This is not overriding UIHandle functions.  `virtual` is right.